### ISSUE TITLE
TIG stack doesn't work with latest docker images - Fixed

### ISSUE THREAD

**Issue by suhaibasaeed** [DataKnox/Cisco-MDT-TIG-Docker#2]
Using the latest docker images as of 7th March 23 I could not get the TIG stack working due to multiple issues. Telegraf failed to connect to influxdb throwing 401 Unauthorized in the logs and Grafana also threw 401 errors when trying to add in influxDB as a data source.

My work around was to use the older versions for container images which were released around the time of the repository creation a few years ago. These versions are listed below:
```
influxdb:1.8.3
grafana:6.3.7
telegraf:1.16.1
```
My working docker-compose.yaml file is also as below:

```
version: "2"
services:
  influxdb:
    container_name: influxdb
    image: influxdb:1.8.3
    ports:
      - "8086:8086"
      - "8083:8083"
    volumes:
      - ./vols/:/var/lib/influxdb
    restart: always

  grafana:
    container_name: grafana
    image: grafana/grafana:6.3.7
    ports:
      - "3000:3000"
    links:
      - influxdb
    restart: always

  telegraf:
    container_name: telegraf
    image: telegraf:1.16.1
    ports:
      - "57000:57000"   
    links:
      - influxdb
    volumes:
      - ./conf/telegraf/telegraf.conf:/etc/telegraf/telegraf.conf:ro
      - /var/run/docker.sock:/var/run/docker.sock
    restart: always
```

It would be greatly appreciated if the repository was updated to reflect this so other people don't have to spend hours troubleshooting it like I did :)

**Comment by DataKnox** [comment]
Should have been merged in #4

**Comment by suhaibasaeed** [comment]
Thanks for the quick turn around @DataKnox, I'm happy to close this one out.
